src/index.js

```diff
@@ -11,7 +11,7 @@ const client = new Discord.Client({
 })
 
 // emoji that goes in the post title
-const tt = '⭐'
+let tt = '⭐'
```

Comment:
We should just get rid of `tt` entirely and replace all uses of it with your new `settings.embedEmoji`